internal/mock/api_backup.gen.go

```diff
@@ -170,31 +170,3 @@ func (mr *MockBackupApiMockRecorder) GetBackupShardIdExecuteWithHttpInfo(arg0 in
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetBackupShardIdExecuteWithHttpInfo", reflect.TypeOf((*MockBackupApi)(nil).GetBackupShardIdExecuteWithHttpInfo), arg0)
 }
-
-// OnlyCloud mocks base method.
-func (m *MockBackupApi) OnlyCloud() api.BackupApi {
-	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "OnlyCloud")
-	ret0, _ := ret[0].(api.BackupApi)
-	return ret0
-}
-
-// OnlyCloud indicates an expected call of OnlyCloud.
-func (mr *MockBackupApiMockRecorder) OnlyCloud() *gomock.Call {
-	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "OnlyCloud", reflect.TypeOf((*MockBackupApi)(nil).OnlyCloud))
-}
-
-// OnlyOSS mocks base method.
-func (m *MockBackupApi) OnlyOSS() api.BackupApi {
-	m.ctrl.T.Helper()
```

Comment:
Not sure, I've updated the openapi submodule and regenerated so that might've brought this change with it (https://github.com/influxdata/influx-cli/commit/d519890f03b11aa52236234e4f4ee4a3c1739565 ?). They aren't used so this  should be fine.